third_party/terraform/resources/resource_google_project_services.go

```diff
@@ -48,330 +54,321 @@ func resourceGoogleProjectServices() *schema.Resource {
 	}
 }
 
-var ignoredProjectServices = []string{"dataproc-control.googleapis.com", "source.googleapis.com", "stackdriverprovisioning.googleapis.com"}
-
-// These services can only be enabled as a side-effect of enabling other services,
-// so don't bother storing them in the config or using them for diffing.
-var ignoreProjectServices = golangSetFromStringSlice(ignoredProjectServices)
-
 func resourceGoogleProjectServicesCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Since `Create` and `Update` are effectively identical, can we use a single `CreateUpdate` method?